src/CMakeLists.txt

```diff
@@ -337,12 +348,20 @@ endif( )
 
 set( CPACK_SOURCE_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}-${CLFFT_VERSION}-${CMAKE_HOST_SYSTEM_NAME}-Source" )
 
+# The following code is setting variables to control the behavior of CPack to generate our
+set( CPACK_PACKAGE_CONTACT "none@gmail.com")
```

Comment:
Is the 'CPACK_PACKAGE_CONTACT' with email required, can you remove this line?

Reply:
It is required for the packaging to succeed, it won't build without it.

Reply:
Looks like the email id is some random one that I like to avoid, can you leave it empty? like
set( CPACK_PACKAGE_CONTACT "")

Reply:
Leaving it with an empty string gives the following error:

```
CMake Error at /usr/share/cmake-3.9/Modules/CPackDeb.cmake:820 (message):
  CPackDeb: Debian package requires a maintainer for a package, set
  CPACK_PACKAGE_CONTACT or CPACK_DEBIAN_PACKAGE_MAINTAINER
Call Stack (most recent call first):
  /usr/share/cmake-3.9/Modules/CPackDeb.cmake:1049 (cpack_deb_prepare_package_vars)
```

Reply:
Does the string have to be an email? Can you try "clFFT maintainers"

Reply:
Indeed, your suggestion makes it work, see the commit below.